tests/E2E/test/campaigns/full/01_orders/05_create_order_without_account.js

```diff
@@ -305,7 +305,11 @@ scenario('Create order by a guest from the Front Office', client => {
   stockCommonScenarios.checkStockProduct(client, productData[0].name + date_time, Menu, Stock, '50', '0', '50');
   stockCommonScenarios.checkStockProduct(client, productData[1].name + date_time, Menu, Stock, '50', '0', '50');
   scenario('Check the movement of the "' + productData[0].name + date_time + '"', client => {
-    test('should go to "Employee" page', () => client.goToSubtabMenuPage(Menu.Configure.AdvancedParameters.advanced_parameters_menu, Menu.Configure.AdvancedParameters.team_submenu));
+    test('should go to "Employee" page', () => {
+      return promise
+        .then(() => client.goToUrl(Menu.Configure.AdvancedParameters.team_submenu + ' a'))
```

Comment:
You should use user navigation instead custom javascript.

Reply:
Done.
Now we close Menu Catalog before opening Advanced Parameters Menu (otherwise selenium will not click with and without scroll)